setup.py

```diff
@@ -0,0 +1,30 @@
+from sqlalchemy import create_engine
+import requests
+from Models.Reviews import Review
+from sqlalchemy.orm import sessionmaker
+URI = "postgres://xeeqkeipvcflil:fead0f891152e8f657d7ee59ba256aa36c24fed629a0ee765b6a6aac6da2610f@ec2-54-225-236-102.compute-1.amazonaws.com:5432/d7mipabothqsm0"
```

Comment:
This is totally :no_good:. Plaintext passwords to real machines on the cloud is a **huge** security problem. Please delete this line and change the password on the machine **immediately**.

@Raphaeljunior